Yosemite/YosemiteTests/Stores/ReceiptStoreTests.swift

```diff
@@ -434,7 +434,7 @@ private extension ReceiptStoreTests {
               coupons: coupons,
               refunds: [],
               fees: fees,
-              taxes: []) // TODO: 5809 - Add makeOrderTaxLine method
+              taxes: [makeOrderTaxLine(totalShippingTax: shippingTax, totalTax: totalTax)])
```

Comment:
How about updating this method to receive `taxes` as input like other properties? I don't think it's necessary to calculate taxes based on `shippingTax` and `totalTax` here, because we are fetching the `OrderTaxLine` array from the API. WDYT?

Reply:
> I don't think it's necessary to calculate taxes 

I agree. I think this will also make tests easier to maintain.
Done in - https://github.com/woocommerce/woocommerce-ios/commit/a0d1c2cc8b2d3cea71d2354150cc5ec5712d9ad9